internal/resources/otelcollector/rbac_test.go

```diff
@@ -296,3 +278,20 @@ func checkClusterRoleBinding(t *testing.T, crb *rbacv1.ClusterRoleBinding, name,
 	require.Equal(t, "ClusterRole", crb.RoleRef.Kind)
 	require.Equal(t, name, crb.RoleRef.Name)
 }
+
+func checkRoleBinding(t *testing.T, rb *rbacv1.RoleBinding, name, namespace string) {
```

Comment:
This function is used only once. I think it is better to keep the checks in the test itself as it was, this makes it easier to follow the test

Reply:
its used in 2 places [here](https://github.com/kyma-project/telemetry-manager/pull/1343/files#diff-8185d89bfd783b46c662f4ebb93b8cd31c9f19984d232b32d49cb833b4084ffbR173) and [here](https://github.com/kyma-project/telemetry-manager/pull/1343/files#diff-8185d89bfd783b46c662f4ebb93b8cd31c9f19984d232b32d49cb833b4084ffbR260) so moved it to a function